packages/vite/src/node/plugins/importAnalysis.ts

```diff
@@ -621,12 +621,17 @@ function transformCjsImport(
     }) as any
   ).body[0] as Node
 
-  if (node.type === 'ImportDeclaration') {
+  if (
+    node.type === 'ImportDeclaration' ||
+    node.type === 'ExportNamedDeclaration'
+  ) {
     if (!node.specifiers.length) {
       return `import "${url}"`
     }
 
     const importNames: ImportNameSpecifier[] = []
+    const exportNames: string[] = []
+    let defaultExports
```

Comment:
This variable needs a typedef cause TS/devs doesn't know what it could potentially be ... leading to implicit any

Reply:
Yes, happy now ready to review